tests/components/zwave_js/test_sensor.py

```diff
@@ -5,10 +5,11 @@
 from zwave_js_server.event import Event
 from zwave_js_server.model.node import Node
 
+from homeassistant.components.binary_sensor import BinarySensorDeviceClass
```

Comment:
I don't think you should be importing BinarySensorDeviceClass inside `test_sensor.py`

Reply:
Agreed, done.